src/main/kotlin/org/openmbee/flexo/mms/routes/ldp/RepoRead.kt

```diff
@@ -18,29 +18,10 @@ private val SPARQL_BGP_REPO: (Boolean, Boolean) -> String = { allRepos, allData
                 mms:etag ?__mms_etag ;
                 mms:org ?$SPARQL_VAR_NAME_ORG ;
                 ${"?repo_p ?repo_o ;" iff allData}
-                .
-                
-            bind(iri(concat(str(?$SPARQL_VAR_NAME_REPO), "/graphs/Metadata")) as ?metadataGraph)
+                .                
         """.reindent(if(allRepos) 3 else 2)}
         ${"}" iff allRepos}
-
-        ${"""
-            optional {
-                ?thing mms:repo ?$SPARQL_VAR_NAME_REPO ; 
-                    ?thing_p ?thing_o ;
-                    .
-            }
-        """.reindent(2) iff allData}
-    }
-        
-    optional {
-        graph ?metadataGraph {
```

Comment:
this will change the semantics of the repo etag

Reply:
that's fine, there's actually another issue about what the etag is representing (more for branches but it applies here too) #162